srcpkgs/ipp-usb/files/README.voidlinux

```diff
@@ -0,0 +1,13 @@
+Void-specific instructions:
+============================
+
+By default, ipp-usb constantly runs in the background. It can also run on
+demand by utilizing udev rules to detect usb printers. You can enable
+this functionality by doing the following:
+
+echo 'OPTS=udev' > /etc/sv/ipp-usb/conf
+touch /etc/sv/ipp-usb/down
+ln -s /etc/sv/ipp-usb /var/services
+cp /usr/share/doc/ipp-usb/71-ipp-usb.rules /etc/udev/rules.d/
```

Comment:
Maybe suggest linking the rule instead, so it will be automatically updated?

Reply:
I have added a suggestion to link the rules, although I don't know if linking from something in /usr/share/doc to something in /etc is the proper way to do it.

Regarding the avahi patch, I have updated the dependency to avahi>=0.8, which does not need the patch.